oss_src/unity/python/sframe/test/test_sarray.py

```diff
@@ -977,6 +977,31 @@ def test_sample(self):
         sa_sample = SArray().sample(.5, 9)
         self.assertEqual(len(sa_sample), 0)
 
+    def test_hash(self):
```

Comment:
Add test that two subsequent calls with seed = None do not have the same values.